src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -384,10 +402,9 @@
         immediate: true
       },
       'specificUserAddress': {
-        handler: throttle(function (value) {
-          if (!value) return
-          this.setFilters()
-          this.getLimits()
+        handler: throttle(async function (value) {
+          await this.setFilters()
+          await this.getLimits()
```

Comment:
`await Promise.all([this.setFilters, this.getLimits()])`

Reply:
These methods should be called one by one, cause `setFilters` gets account ID by user email asynchronously, and `getLimits` uses that account ID